usaspending_api/database_scripts/matview_generator/tas_search_matview.json

```diff
@@ -2,7 +2,7 @@
   "final_name": "tas_search_matview",
   "refresh": true,
   "matview_sql": [
-    "select distinct",
+    "select",
```

Comment:
Why isn't this nessicary? I thought that that was a key point of why the min function was okay.

Reply:
The grouping/`min` achieves (roughly) the same thing as the `distinct` in this case so the `distinct` is superfluous.

Reply:
that makes sense